sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs

```diff
@@ -105,26 +105,29 @@ public virtual async Task<TrainingOperation> StartTrainingAsync(Uri trainingFile
         }
 
         /// <summary>
+        /// Get detailed information about a custom model.
```

Comment:
```suggestion
        /// Get a description of a custom model, including the types of forms it can recognize and the fields it will extract for each form type.
```